src/cli.ts

```diff
@@ -254,10 +292,47 @@ program
   )
   .option('--url <url>', 'Kibana URL to fetch all public and private locations')
   .addOption(auth)
+  .option(
```

Comment:
Would be good to move these common options to options.ts and reuse it for push/locations.